CONTRIBUTING.md

```diff
@@ -129,27 +146,33 @@ Before you submit your pull request consider the following guidelines:
 * Push your branch to GitHub:
 
     ```shell
-    git push origin my-fix-branch
+    git push <remote> my-fix-branch
```

Comment:
Why not `origin`. I'm afraid `<remote>` will confuse git beginners (and the instructions do not matter for experiences users, because they already know their way around).

Reply:
Right, origin is better. I was thinking of my setup where angular/angular.js is the origin, and my fork just a remote.
Technically, since git now uses "match" as the default push strategy, should git push be enough?

Reply:
I think that for branches created in the way we show here, you need to have set the upstream once before you can just `git push`. I.e. you need to have run `git push --set-upstream origin my-fix-branch` the first time.